internal/attestation/snp/validator.go

```diff
@@ -26,42 +26,45 @@ type Validator struct {
 	validateOpts *validate.Options
 	reportSetter attestation.ReportSetter
 	logger       *slog.Logger
+	name         string
 }
 
 // NewValidator returns a new Validator.
-func NewValidator(VerifyOpts *verify.Options, ValidateOpts *validate.Options, log *slog.Logger) *Validator {
+func NewValidator(VerifyOpts *verify.Options, ValidateOpts *validate.Options, log *slog.Logger, name string) *Validator {
 	return &Validator{
 		verifyOpts:   VerifyOpts,
 		validateOpts: ValidateOpts,
 		logger:       log,
+		name:         name,
 	}
 }
 
 // NewValidatorWithReportSetter returns a new Validator with a report setter.
 func NewValidatorWithReportSetter(VerifyOpts *verify.Options, ValidateOpts *validate.Options,
-	log *slog.Logger, reportSetter attestation.ReportSetter,
+	log *slog.Logger, reportSetter attestation.ReportSetter, name string,
 ) *Validator {
 	return &Validator{
 		verifyOpts:   VerifyOpts,
 		validateOpts: ValidateOpts,
 		reportSetter: reportSetter,
 		logger:       log,
+		name:         name,
 	}
 }
 
-// OID returns the OID of the validator.
+// OID returns the root OID for the raw SNP report extension used by the validator.
```

Comment:
Why `root`? And, if you are changing the docstring here, please also change it for TDX.

Reply:
https://github.com/edgelesssys/contrast/blob/main/internal/oid/oid.go

Reply:
Hm. I'd say it's technically a node, but not the root node.